src/components/shapediver/ui/ParametersAndExportsAccordionComponent.tsx

```diff
@@ -125,33 +136,50 @@ export default function ParametersAndExportsAccordionComponent({ parameters, exp
 	const elements: JSX.Element[] = [];
 
 	// loop through the created elementGroups to add them
-	for (const e in elementGroups) {
-		const g = elementGroups[e];
-
+	let accordionItems: JSX.Element[] = [];
+	for (const g of elementGroups) {
+	
 		// add dividers between the elements
 		const groupElements: JSX.Element[] = [];
 		g.elements.forEach((element, index) => {
 			groupElements.push(element);
 			if (index !== g.elements.length - 1) groupElements.push(<Divider key={element.key + "_divider"} my="sm" />);
 		});
 
-		// create an Accordion.Item for each element
+		if (g.group && (!avoidSingleComponentGroups || g.elements.length > 1)) {
+			accordionItems.push(
+				<Accordion.Item key={g.group.id} value={g.group.id}>
+					<Accordion.Control>{g.group.name}</Accordion.Control>
+					<Accordion.Panel key={g.group.id}>
+						{groupElements}
+					</Accordion.Panel>
+				</Accordion.Item>
+			);
+		}
+		else {
+			if (accordionItems.length > 0) {
+				elements.push(
+					<Accordion variant="contained" radius="md" className={classes.container} key={accordionItems[0].key}>
+						{ accordionItems }
+					</Accordion>
+				);
+				accordionItems = [];
+			}
+			elements.push(groupElements[0]);
```

Comment:
@LemToUp could you continue this PR, and wrap the `groupElements[0]` inside a Mantine [Paper component](https://mantine.dev/core/paper/ ) such that the layout problem shown in my [main comment](https://github.com/shapediver/ShapeDiverReactExample/pull/46#issue-2093935846) is fixed?

Reply:
@snabela https://github.com/shapediver/ShapeDiverReactExample/pull/46/commits/e4af57b9c6d88715d616132c7726f20387ea70a2